static/app/views/settings/organizationIntegrations/integrationReposAddRepository.tsx

```diff
@@ -58,6 +58,7 @@ export function IntegrationReposAddRepository({
     },
     retry: 0,
     staleTime: 20_000,
+    placeholderData: previousData => (debouncedSearch ? previousData : undefined),
```

Comment:
make sure that we keep filtered items on screen while searching to avoid layout shift, however, if we have no search term, we’d want to fall back to the empty items so that we can see the empty message